### PR TITLE
AlertService unit tests

### PR DESCRIPTION
**Pull request by viikufa**
* returned old name for realm models of RuuviTagRealm
* implemented tests with handling post notifications

### CODE REVIEW

File: station/Classes/Model/Abstractions/RuuviTagProtocol.swift [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    var version: Int { get }
+    var humidity: Double? { get }
+    var pressure: Double? { get }
+    var inHg: Double? { get }
+    var mmHg: Double? { get }
+    var celsius: Double? { get }
+    var mac: String? { get }
+    var fahrenheit: Double? { get }
+    var kelvin: Double? { get }
+    var isConnected: Bool { get }
```
Comment (rinat-enikeev) [inline station/Classes/Model/Abstractions/RuuviTagProtocol.swift]: nit: could you please arrange props in logical (starting from uuid) or even better alphabetical order.

File: stationTests/Services/Alert/AlertServiceSpec.swift [hunk 2]
```diff
@@ -0,0 +1,678 @@
...
+            }
+        }
+        // MARK: - Temperature
+        describe("Temperature") {
+            context("when set lower") {
+                it("must return lower") {
+                    alertService.setLower(celsius: randomDouble, for: uuid)
+                    expect(alertService.lowerCelsius(for: uuid)).to(equal(randomDouble))
+                }
+                it("if upper temp is setted must send notification AlertServiceAlertDidChange with type .temperature(lower: l, upper: u)") {
```
Comment (rinat-enikeev) [inline stationTests/Services/Alert/AlertServiceSpec.swift]: nit: just `set`, https://www.quora.com/Is-setted-a-wrong-word

File: stationTests/Services/Alert/AlertServiceSpec.swift [hunk 3]
```diff
@@ -0,0 +1,678 @@
...
+                    alertService.alertPersistence.setMovement(counter: randomInt, for: uuid)
+                    expect {
+                        alertService.setMovement(description: uuid, for: uuid)
+                    }.toEventually(
+                        postNotifications(self.equalName(.AlertServiceAlertDidChange, for: uuid)
+                    ), timeout: self.notificationWaitTimeout)
+                }
+            }
+        }
+        // MARK: - HeartBit
```
Comment (rinat-enikeev) [inline stationTests/Services/Alert/AlertServiceSpec.swift]: nit: Heartbeat

File: stationTests/Services/Alert/AlertServiceSpec.swift [hunk 4]
```diff
@@ -0,0 +1,678 @@
...
+                    expect(fakeDelegate.uuid).toEventually(equal(uuid))
+                    expect(fakeDelegate.service).toEventuallyNot(beNil())
+                    expect(localNotificationManager.uuid).toEventually(equal(uuid))
+
+                    expect(localNotificationManager.reason).toEventually(equal(LowHighNotificationReason.high))
+                    expect(localNotificationManager.type).toEventually(equal(LowHighNotificationType.pressure))
+                }
+            }
+        }
+        // MARK: - HeartBit
```
Comment (rinat-enikeev) [inline stationTests/Services/Alert/AlertServiceSpec.swift]: nit: WPSData?

File: station/Classes/Business/Services/WeatherProvider/WPSData.swift [hunk 5]
```diff
@@ -0,0 +1,23 @@
+//
+//  WPSData.swift
+//  station
+//
+//  Created by Viik.ufa on 13.03.2020.
+//  Copyright © 2020 Ruuvi Innovations Oy. All rights reserved.
```
Comment (ojousima) [inline station/Classes/Business/Services/WeatherProvider/WPSData.swift]: BSD-3 license

File: station/Classes/Model/Abstractions/RuuviTagProtocol.swift [hunk 6]
```diff
@@ -0,0 +1,34 @@
+//
+//  RuuviTagProtocol.swift
+//  station
+//
+//  Created by Viik.ufa on 14.03.2020.
+//  Copyright © 2020 Ruuvi Innovations Oy. All rights reserved.
```
Comment (ojousima) [inline station/Classes/Model/Abstractions/RuuviTagProtocol.swift]: BSD-3 license

File: station/Classes/Model/Realm/Abstraction/RuuviTagRealmProtocol.swift [hunk 7]
```diff
@@ -0,0 +1,22 @@
+//
+//  RuuviTagRealm.swift
+//  station
+//
+//  Created by Viik.ufa on 14.03.2020.
+//  Copyright © 2020 Ruuvi Innovations Oy. All rights reserved.
```
Comment (ojousima) [inline station/Classes/Model/Realm/Abstraction/RuuviTagRealmProtocol.swift]: BSD-3 license

File: stationTests/MockAlertPersistence.swift [hunk 8]
```diff
@@ -0,0 +1,89 @@
+//
+//  MockAlertPersistence.swift
+//  stationTests
+//
+//  Created by Viik.ufa on 13.03.2020.
+//  Copyright © 2020 Ruuvi Innovations Oy. All rights reserved.
```
Comment (ojousima) [inline stationTests/MockAlertPersistence.swift]: BSD-3 license

File: stationTests/MockAlertServiceObserver.swift [hunk 9]
```diff
@@ -0,0 +1,22 @@
+//
+//  MockAlertServiceObserver.swift
+//  stationTests
+//
+//  Created by Viik.ufa on 15.03.2020.
+//  Copyright © 2020 Ruuvi Innovations Oy. All rights reserved.
```
Comment (ojousima) [inline stationTests/MockAlertServiceObserver.swift]: BSD-3 license

File: stationTests/MockCalibrationService.swift [hunk 10]
```diff
@@ -0,0 +1,26 @@
+//
+//  MockCalibrationService.swift
+//  stationTests
+//
+//  Created by Viik.ufa on 14.03.2020.
+//  Copyright © 2020 Ruuvi Innovations Oy. All rights reserved.
```
Comment (ojousima) [inline stationTests/MockCalibrationService.swift]: BSD-3 license

File: stationTests/MockLocalNotificationsManager.swift [hunk 11]
```diff
@@ -0,0 +1,30 @@
+//
+//  MockLocalNotificationsManager.swift
+//  stationTests
+//
+//  Created by Viik.ufa on 14.03.2020.
+//  Copyright © 2020 Ruuvi Innovations Oy. All rights reserved.
```
Comment (ojousima) [inline stationTests/MockLocalNotificationsManager.swift]: BSD-3 license

File: stationTests/MockRuuviTag.swift [hunk 12]
```diff
@@ -0,0 +1,33 @@
+//
+//  MockRuuviTag.swift
+//  stationTests
+//
+//  Created by Viik.ufa on 14.03.2020.
+//  Copyright © 2020 Ruuvi Innovations Oy. All rights reserved.
```
Comment (ojousima) [inline stationTests/MockRuuviTag.swift]: BSD-3 license

File: stationTests/Services/Alert/AlertServiceSpec.swift [hunk 13]
```diff
@@ -0,0 +1,678 @@
+//
+//  AlertServiceSpec.swift
+//  stationTests
+//
+//  Created by Viik.ufa on 13.03.2020.
+//  Copyright © 2020 Ruuvi Innovations Oy. All rights reserved.
```
Comment (ojousima) [inline stationTests/Services/Alert/AlertServiceSpec.swift]: BSD-3 license

**Pull request merged**